app/views/teacher_interface/further_information_request_items/edit.html.erb

```diff
@@ -17,5 +17,25 @@
     <p class="govuk-body">You can upload this document on the next screen.</p>
   <% end %>
 
+  <% if @further_information_request_item.work_history_contact? %>
+    <% @work_history = @further_information_request_item.work_history %>
+      <p class="govuk-body">Enter the contact details of a suitable referee who can verify your time spent working at <%= @work_history.school_name %> from <%= @work_history.start_date.to_fs(:month_and_year) %> <% if @work_history.end_date.present? %>
+          to <%= @work_history.end_date.to_fs(:month_and_year) %>
+        <% end %></p>
+      <%= f.govuk_text_field :contact_name %>
+
+      <%= f.govuk_text_field :contact_job %>
+
+      <%= f.govuk_email_field :contact_email %>
+    <%= govuk_details(summary_text: "Choosing someone to verify this role") do %>
```

Comment:
Looks like the indentation in this file is a little off, the fields should be at the same level of the details component I think.